### PR TITLE
MS-241: Session details, Interview details. 

### PR DESCRIPTION
**Pull request by valeferaudo**

### CODE REVIEW

File: src/Components/Admin/Interviews/interviews.module.css [hunk 1]
```diff
@@ -1,71 +1,128 @@
+table {
```
Comment (sofpag) [inline src/Components/Admin/Interviews/interviews.module.css:1]: acá

File: src/Components/Admin/Interviews/interviews.module.css [hunk 2]
```diff
@@ -1,71 +1,128 @@
...
-
-.table td:last-child,
-.table th:last-child {
-  border-radius: 0 20px 20px 0;
-  border-right: 2px solid #002147;
+.blueBtn:hover {
+  background-color: #680404;
+  cursor: pointer;
 }
+th{
```
Comment (sofpag) [inline src/Components/Admin/Interviews/interviews.module.css:126]: acá

File: src/Components/Admin/Sessions/Session/session.module.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    margin-right: 40px;
+    font: normal bold 20px Century Gothic;
+    color: #9b0000;
+  }
+  .profileName {
+    font: normal normal normal 20px/27px Century Gothic;
+    color: #091f40;
+    width: 50%;
+  }
+  tr > td {
```
Comment (sofpag) [inline src/Components/Admin/Sessions/Session/session.module.css:94]: acá

File: src/Components/Admin/Sessions/sessions.module.css [hunk 4]
```diff
@@ -1,71 +1,128 @@
+table {
```
Comment (sofpag) [inline src/Components/Admin/Sessions/sessions.module.css:1]: acá

File: src/Components/Admin/Sessions/sessions.module.css [hunk 5]
```diff
@@ -1,71 +1,128 @@
...
-
-.table td:last-child,
-.table th:last-child {
-  border-radius: 0 20px 20px 0;
-  border-right: 2px solid #002147;
+.blueBtn:hover {
+  background-color: #680404;
+  cursor: pointer;
 }
+th{
```
Comment (sofpag) [inline src/Components/Admin/Sessions/sessions.module.css:126]: acá

**Review by sofpag — APPROVED** [review]

**Review by julianv97 — CHANGES_REQUESTED** [review]
Screen Shot 2022-01-15 at 19 28 28
we need to fix this styles, in sessions same
Screen Shot 2022-01-15 at 19 30 23

**Review by julianv97 — APPROVED** [review]

**Pull request merged**